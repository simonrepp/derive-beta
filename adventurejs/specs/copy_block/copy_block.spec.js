const fs = require('fs');
const path = require('path');

const { parse, AdventureParseError, AdventureValidationError } = require('../../adventure.js');

const input = fs.readFileSync(path.join(__dirname, 'copy_block.ad'), 'utf-8');

describe('Copying a block', () => {
  test('succeeds', () => {
    const document = parse(input);
    expect(document.raw()).toMatchSnapshot();
  });
});
