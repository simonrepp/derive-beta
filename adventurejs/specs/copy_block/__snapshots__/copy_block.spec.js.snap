// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copying a block succeeds 1`] = `
Array [
  Object {
    "Block": "Veni
Vidi
Vici",
  },
  Object {
    "Copied Block": "Veni
Vidi
Vici",
  },
  Object {
    "Copied & Modified Block": "Veni
Vidi
Vici
That is latin I think.",
  },
]
`;
